lib/index.js

```diff
@@ -43,19 +43,15 @@ function Gpm(name, options) {
       , is_dev: false
     });
 
+    if (typeof(packages) === "string") {
```

Comment:
Can you please change this to `typeof packages === "string"`? I have never used `typeof` as function.
Maybe I have to add that to code stlye.